docs/modules/ROOT/pages/how-tos/workflows/keep-remote-pipelines-up-to-date.adoc

```diff
@@ -0,0 +1,107 @@
+= Maintaining remote Tekton pipelines up to date
```

Comment:
General comment on structure of this page. If we create a structure similar to the git submodule approach, I think that this page would have sections like:

(brief description of the problem we are trying to solve)
* Implementing this guideline
  * Basic definitions
  * Create a reference component (see comment about naming the component)
  * Configure renovate
  * Configure additional components with the remote resolver
* Benefits of this guideline
* Potential drawbacks

Reply:
This change required a hard refactor of the documentation. Please re-review it.